v63002/u_inref/gtm8711.csh

```diff
@@ -0,0 +1,21 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests GDE appropriately maintains the return status when invoked from the shell
+#
```

Comment:
The currently implemented test case invokes GDE and inside the GDE prompt sends an invalid command "asdfasdf". Would additionally like a test of "GDE asdfsdf" specified at the shell prompt and verify $status. Should be the same status as the first test.

Reply:
The below comment is the only one remaining to be addressed before I can approve this pull request.

*The currently implemented test case invokes GDE and inside the GDE prompt sends an invalid command "asdfasdf". Would additionally like a test of "GDE asdfsdf" specified at the shell prompt and verify $status. Should be the same status as the first test.*